postgres/tests/test_unit.py

```diff
@@ -13,8 +13,8 @@
 
 
 @pytest.fixture
-def check():
-    check = PostgreSql('postgres', {}, {})
+def check(pg_instance):
```

Comment:
Does this work?

Reply:
Yes, I needed to pass a list of instances to the check so the constructor could check if `custom_metrics` was defined and subsequently throw a deprecation notice.

However, I now just check to see if the list of instances is not `None` before I check to see if `custom_metrics` was defined. This occurs here: https://github.com/DataDog/integrations-core/pull/2043/files#diff-5557a2bc9bebe6d84b8a6fa72246b83bR331